udata/templates/home.html

```diff
@@ -92,7 +92,7 @@ <h1 class="page-header">
                                 <h3 class="page-header">{{ _('Used datasets') }}</h3>
                             </div>
 
-                            {% for dataset in reuse.datasets[:4] %}
+                            {% for dataset in (reuse.datasets|rejectattr('deleted')|list)[:4] %}
```

Comment:
It's just one [builtin filter](http://jinja.pocoo.org/docs/2.9/templates/#rejectattr), I don't get what's bothering you, it's meant for that, no ?